server/chat.ts

```diff
@@ -2054,6 +2064,11 @@ export const Chat = new class {
 				targetUser.send(`${prefix}The user '${Utils.escapeHTML(user.name)}' attempted to challenge you to a battle but was blocked. To enable challenges, use /unblockchallenges ${options}`);
 				targetUser.notified.blockChallenges = true;
 			}
+		} else if (blocked === 'invite') {
+			if (!targetUser.notified.blockChallenges) {
```

Comment:
Did you plan to use the same notification system as `blockChallenges` or a different one?

If "same", you should remove `notified.blockInvites` from `users.ts`. If different, this line should be `notified.blockInvites`.